### PR TITLE
Feat/add missing licenses

### PR DESCRIPTION
**Pull request by jdsika**
Wait for the axios PR is merged to get a proper diff

### CODE REVIEW

File: packages/siwt.xyz/pages/index.module.css [hunk 1]
```diff
@@ -1,2 +0,0 @@
-.page {
```
Comment (jdsika) [inline packages/siwt.xyz/pages/index.module.css:1]: Empty file?

File: packages/smart-contracts/src/ligo/src/core/ledger.jsligo [hunk 2]
```diff
@@ -1,12 +1,10 @@
-#import "./errors.jsligo" "FA2_Errors"
-
 /*
-export namespace Types {
-   type token_id = nat;
```
Comment (jdsika) [inline packages/smart-contracts/src/ligo/src/core/ledger.jsligo:5]: merge conflict?

File: packages/discord/src/app/modules/api/api.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ import { getMember } from '../../common/discord/utils'
 import { accessControlQuery } from '../../config/config'
 
 const app = express()
-const port = process.env.PORT || 3000
```
[the remark this answers is not in the document]
Comment (royscheeren) [reply packages/discord/src/app/modules/api/api.ts:18]: Yes

File: packages/siwt.xyz/pages/index.module.css (hunk 1 above)
Comment (royscheeren) [reply packages/siwt.xyz/pages/index.module.css:1]: Deleted file

File: packages/smart-contracts/src/ligo/src/core/ledger.jsligo (hunk 2 above)
Comment (royscheeren) [reply packages/smart-contracts/src/ligo/src/core/ledger.jsligo:5]: Nope, this type was not used

File: packages/smart-contracts/src/ligo/src/core/operators.jsligo [hunk 4]
```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2022, vDL Digital Ventures GmbH <info@vdl.digital>
```
[the remark this answers is not in the document]
Comment (jdsika) [reply packages/smart-contracts/src/ligo/src/core/operators.jsligo:2]: In next PR #34 : Copyright (c) 2021 Pierre-Emmanuel Wulfman, Sander Spies, Melwyn Saldanha and ligolang.

**Review by royscheeren — APPROVED** [review]

**Pull request merged**